playbooks/openshift/delete-cluster.yml

```diff
@@ -4,14 +4,28 @@
 #  roles:
 #  - ../../galaxy/openshift-ansible-contrib/roles/openshift-pv-cleanup
 
+- vars_prompt:
```

Comment:
The playbooks will need a `hosts` line, otherwise it will error out. Please test to ensure the playbook runs.